config/config.exs

```diff
@@ -25,6 +25,10 @@ config :skate, SkateWeb.Endpoint,
   render_errors: [view: SkateWeb.ErrorView, accepts: ~w(html json)],
   pubsub: [name: Skate.PubSub, adapter: Phoenix.PubSub.PG2]
 
+config :skate, SkateWeb.AuthManager,
+  issuer: "skate",
+  secret_key: "test key"
```

Comment:
Instead of having this as the default, should this be specified in `config/test.exs`? What's the key used for, what value should it have in dev, and would you ever want to set it by env variable in dev?

Similar for the `Ueberauth.Strategy.Fake` below. If dev and test and prod need to be different, we might want to specify them in the environment-specific configuration so there's less chance of the insecure configuration accidentally applying in prod.